libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/addAttributeFilter/tests/addAttributeFilterUtils.test.ts

```diff
@@ -1,59 +1,86 @@
 // (C) 2023 GoodData Corporation
 
-import { IInsight } from "@gooddata/sdk-model";
-import { VisType } from "@gooddata/sdk-ui";
 import { isLocationIconEnabled } from "../addAttributeFilterUtils";
+import { newInsightMap } from "../../../../../_staging/metadata/objRefMap";
 
 describe("add attribute filter utils", () => {
     describe("isLocationIconEnabled", () => {
-        const mockInsight: IInsight = {
+        const createMockInsight = (type: string, id: string): any => ({
```

Comment:
Isn't it possible to type this? `any` mocks in tests are often source of issues in the future when you change the original type and then you will not notice that tests no longer works or just tests something else.

Reply:
I used the type "any" for quick testing purposes and forgot to update it before committing the code. I have now updated it to use the appropriate and specific type. 

Thank you for bringing this issue to my attention!